tests/python/tank_test/tank_test_base.py

```diff
@@ -517,12 +610,12 @@ def setup_multi_root_fixtures(self):
                                     "code": self.primary_root_name,
                                     "windows_path": self.tank_temp,
                                     "linux_path": self.tank_temp,
-                                    "mac_path": self.tank_temp }
+                                    "mac_path": self.tank_temp}
 
             self.add_to_sg_mock_db(self.primary_storage)
 
-        self.setup_fixtures(parameters = {"core": "core.override/multi_root_core",
-                                          "skip_template_reload": True})
+        self._setup_fixtures(parameters={"core": "core.override/multi_root_core",
```

Comment:
Same question about using `**kwargs` vs a `parameters` dict. The former would be more pythonic it seems?